src/gmt_api.c

```diff
@@ -1609,15 +1612,15 @@ GMT_LOCAL char ** gmtapi_process_keys (void *V_API, const char *string, char typ
 					change_type = false;
 					for (kk = 3; s[k][kk]; kk++) {	/* Examine characters in the modifier string */
 						if (strchr ("-+", s[k][kk])) {	/* Start of all (+) versus at least one (-) */
-							kase = (s[k][kk] == '-') ? 0 : 1;	/* Set kase and go to next letter */
+							kase = (s[k][kk] == '-') ? K_OR : K_AND;	/* Set kase and go to next letter */
 							continue;
 						}
 						count[kase]++;	/* How many AND and how many OR modifiers (depending on kase) */
 						modifier[1] = s[k][kk];	/* Set current modifier */
 						if (strstr (opt->arg, modifier)) given[kase]++;	/* Match found with given option */
 					}
 					/* Only change the key if we found all the AND modifiers and at least one of the OR modifiers (if any were given) */
-					if ((count[0] == 0 || (count[0] && given[0])) && count[1] == given[1]) change_type = true;
+					if ((count[K_OR] == 0 || (count[K_OR] && given[K_OR])) && count[K_AND] == given[K_AND]) change_type = true;
```

Comment:
I find that these makes it harder to follow the code. One will never know what is ``K_OR`` without going search code somewhere else.

Reply:
IN my case it was the opposite: 0 and 1 were meaningless but now I know if it is boolean OR or AND we are doing with the modifiers.  But I agree it is hard to follow regardless.